newlib/libc/sys/vita/fs.c

```diff
@@ -50,3 +56,38 @@ int rmdir(const char *pathname)
 	errno = 0;
 	return 0;
 }
+
+int fsync (int __fd)
+{
+	int ret;
+
+	DescriptorTranslation *fdmap = __vita_fd_grab(__fd);
+
+	if (!fdmap) {
+		errno = EBADF;
+		return -1;
+	}
+
+	switch (fdmap->type)
+	{
+	case VITA_DESCRIPTOR_FILE:
+		ret = sceIoSyncByFd(fdmap->sce_uid, 0);
+		break;
+	case VITA_DESCRIPTOR_TTY:
+	case VITA_DESCRIPTOR_SOCKET:
+	case VITA_DESCRIPTOR_DIR:
```

Comment:
if you remove `VITA_DESCRIPTOR_DIR` in here then move it to #63 
i'll merge it first.